### PR TITLE
allow private git submodules cloning

### PR DESCRIPTION
**Pull request by ab77**
change-type: patch

### CODE REVIEW

File: .github/workflows/renovate.yml [hunk 1]
```diff
@@ -56,5 +56,9 @@ jobs:
                 "matchHost": "ghcr.io",
                 "username": "${{ github.actor }}",
                 "password": "${{ secrets.GITHUB_TOKEN }}"
+              },
+              {
+                "hostType": "github",
```
Comment (klutchell) [inline .github/workflows/renovate.yml:61]: Do we not also need `"matchHost": "api.github.com"` here?

Comment (ab77) [reply .github/workflows/renovate.yml:61]: It's not really clear, some examples use `"matchHost": "api.github.com"`, some `"github.com"`. I imagine it's a regex anyway, and probably defaults to `githib.com` when `"hostType": "github"`, but we can add it in for completeness I guess.

Comment (klutchell) [reply .github/workflows/renovate.yml:61]: From the docs

> The token from the hostRules entry matching hostType=github and matchHost=api.github.com is added as the default authentication for github.com. For those running against github.com, this token will be the default platform token.

it's not clear if it means we need to include it or not, but it wouldn't hurt

**Comment by klutchell** [comment]
LGTM

**Pull request merged**